source/frontends/libretro/game.h

```diff
@@ -46,6 +47,9 @@ namespace ra2
 
     common2::Speed mySpeed;  // fixed speed
 
+    std::chrono::steady_clock::time_point first_btn_press; // for quit
```

Comment:
keep same style as rest of this file, which is camel case.

Reply:
Applied.